src/main/java/mate/academy/bookstore/service/BookServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.bookstore.service;
+
+import java.util.List;
+import java.util.Random;
+import lombok.RequiredArgsConstructor;
+import mate.academy.bookstore.dto.BookDto;
+import mate.academy.bookstore.dto.CreateBookRequestDto;
+import mate.academy.bookstore.mapper.BookMapper;
+import mate.academy.bookstore.model.Book;
+import mate.academy.bookstore.repository.BookRepository;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class BookServiceImpl implements BookService {
+    private final BookRepository bookRepository;
+    private final BookMapper bookMapper;
+
+    @Override
+    public BookDto save(CreateBookRequestDto requestDto) {
+        Book book = bookMapper.toModel(requestDto);
+        book.setIsbn("abcd " + new Random().nextInt(1000));
```

Comment:
magic numbers/values

Reply:
Not resolved. Additionally, by task requirements, ISBN should be passed to the API by client, not generated.